New Teacher Info Layout

Reimirno opened the pull request:
I copied over the original code to ChatGPT and said "This is a rail partial displaying Teachar info styled with Bootstrap. I don't like it so Make it into two-column layout." And boom.

What was generated made a lot of sense. I read through carefully but I only tweaked a little bit... it is simple and responsive design.

Of course I proceed to add into more fields from Teacher table loll. I add in the "third column button" design too (of course this is subject to change, just a show of concept; I highly doubt the copy button would be useful at all lol). Now it looks like this:

![image](https://user-images.githubusercontent.com/82516689/229319196-9bd17e4c-ba06-4ce7-a9f8-9a586f79efa0.png)

cycomachead commented on app/views/teachers/_teacher_info.html.erb:
```diff
@@ -1,68 +1,106 @@
-      <div class="col-sm-4">
-        <h5>Snap<i>!</i> Username</h5>
+      <div class="col-sm-2">
+        <button class="btn btn-primary" onclick="copyToClipboard('<%= teacher.email %>')">Copy</button>
```
Can we use a `btn-sm` or `btn-xs` class? And instead of copy, I think We should have font awesome, and I think we should be able to use `fa-copy`

cycomachead reviewed (approved):
Nice

Couple minor UI tweaks:
* can we remove the text center, kinda looks weird to me
* For the IP addresses, can we call `to_sentence` or join by "," or whatever is easiest?

Looks good nice use of chatgpt

Reimirno commented:
Made all changes as per requested:
1. Changed copy button to `btn-sm` (xs is too small).
2. Replaced the copy button text with an fa copy icon.
3. Removed all text-center (yes it looks less weird now).
4. Display ip history with `to_sentence`.
5. Add fallback of "-" for ip history and last session.
